governance-and-automation/provisioning/templates.md

```diff
@@ -1,8 +1,8 @@
 ---
 description: >-
-  Syskit Point templates help you define which types of workspaces your
-  end-users can request and create. Each template contains properties and rules
-  that guide the creation process and ensure all works
+ Syskit Point templates help you define which types of workspaces your
+ end-users can request and create. Each template contains properties and rules
+ that guide the creation process and ensure all works
```

Comment:
that guide the workspace creation process.
and ensure all works se može maknuti